app/views/topics/show.html.erb

```diff
@@ -2,6 +2,7 @@
   <h1>Topic:</h1>
 
   <h2> <%= @topic.name %></h2>
+  <h3> Author: <%= @topic.user.username %> </h3>
```

Comment:
This isn't a header from a semantic perspective. Use a `<p>` tag or `<span>` or `<div>`